docs/documentation/dependent-resources.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Dependent Resources Feature description: Dependent Resources Feature layout: docs permalink:
```

Comment:
Unfortunatelly formatter usually brokes this header of the file, just make sure that after format this part is reverted.

Reply:
Ah yes, I did it several times while writing the docs but I guess I need to deactivate automatic formatting on commits in my IDE.